app/Http/Controllers/PatientController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\PatientRequest;
+use App\Http\Requests\UpdatePatientRequest;
+use App\Models\Patient;
+use App\Http\Resources\PatientResource;
+
+class PatientController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $patient = Patient::all();
+        return $patient;
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\PatientRequest  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(PatientRequest $request)
+    {
+        $data = $request->validated();
+        
+        $patient = new Patient;
+        $patient->fill($data);
+        $patient->save();
+
+        return new PatientResource($patient);
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $patient = Patient::where('id', $id)->get();
```

Comment:
We can use the Patient model in route model binding and inject a patient model in function.
This statement $patient = Patient::where('id', $id)->get(); can be converted into following.
$patient = Patient::find($id);